src/scss/header/_navegacion.scss

```diff
@@ -1,23 +1,60 @@
-.header-content nav {
-  display: flex;
-}
+@use "base/variables" as v;
+@use "base/mixins" as m;
 
-.header-content nav > ul {
+.menu-icon {
   display: none;
-  color: #344563;
-}
-
-.header-content nav > ul > * {
   margin-right: 0.8rem;
-  padding: 1rem 1.2rem;
-}
 
-.header-content nav > ul > *:last-child {
-  margin-right: 0;
+  a {
+    display: block;
+    padding: 1rem 1.2rem;
+    color: #344563;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &:hover {
+    scale: 1.3;
+    transition-duration: 0.3s;
+    color: #6070ff;
+  }
+
+  @include m.tablet {
+    display: block;
+  }
 }
 
-.menu-icon:hover {
-  transform: scale(1.3);
-  transition-duration: 0.3s;
-  color: #6070ff;
+// Trying to create the cover
+.menu.reveal {
+  display: block;
+  position: absolute;
+  height: 100vh;
+  width: 100vw;
+  right: 0;
+  top: 0;
+  background-color: #6070ffed;
+
+  ul {
+    width: 90%;
+    margin: 0 auto;
+    margin-top: 4.8rem;
+
+    .menu-icon {
+      display: block;
+      margin: 0;
+
+      a {
+        color: #fff;
+        font-weight: 600;
+        font-size: 3.2rem;
+      }
+
+      &:hover {
+        background-color: #8291ff;
+        scale: 1.03;
+      }
+    }
+  }
 }
```

Comment:
- [ ] I think it would be better if you update your mobile menu design in order it follows the design from [your chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Right now, you can see that it misses the `blur effect` on the **background**.

| Figma Design | Your Design |
| -------------------- | ---------------- |
| ![Screenshot from 2023-01-04 13-57-26](https://user-images.githubusercontent.com/83042742/223242457-5b5a8c30-dd8e-4923-878d-4ab4eadb7d13.png) | ![Screenshot from 2023-03-07 18-57-31](https://user-images.githubusercontent.com/83042742/223494774-3a6874a6-a237-4572-9f8f-09f2c33ce115.png) |